### ISSUE TITLE
[1.10.2][Forgelin 1.3.0] Crash -- java.lang.NoClassDefFoundError: kotlin/TypeCastException

### ISSUE THREAD

**Issue by addrever** [shadowfacts/Forgelin#7]
I am getting a crash when I use the latest version of Forgelin.  I have attached the crashlog.  If I downgrade to 1.2.0 the error goes away.  I have attached my crash report and can provide any other information you require.  I'm using forge 2221 for Minecraft 1.10.2

[crash-2017-02-16_06.31.43-client.txt](https://github.com/shadowfacts/Forgelin/files/780487/crash-2017-02-16_06.31.43-client.txt)

**Comment by shadowfacts** [comment]
Umm, that should be impossible. Try redownloading Forgelin from CurseForge.

**Comment by Darkosto** [comment]
Same crash here: 

http://pastebin.com/Ae6DFLAg

**Comment by JamiesWhiteShirt** [comment]
[Similar issue reported here](https://bitbucket.org/jamieswhiteshirt/literal-ascension/issues/29/1102-can-not-find-forgelin-when-using).

**Comment by Morpheus1101** [comment]
FL 1.3.0 Causes this issue in the MMD-Dev pack, reverting to 1.2.0 solves the issue


[08:58:52] [Server thread/ERROR] [LibrarianLib]: **********************************************************************************
[08:58:52] [Server thread/ERROR] [LibrarianLib]:         Shadowfacts' Forgelin must be installed for LibrarianLib to work!
[08:58:52] [Server thread/ERROR] [LibrarianLib]:  Download it here: https://minecraft.curseforge.com/projects/shadowfacts-forgelin
[08:58:52] [Server thread/ERROR] [LibrarianLib]: **********************************************************************************
[08:58:52] [Server Shutdown Thread/INFO]: Stopping server
> 2017-02-17 08:58:52,191 ERROR Attempted to append to non-started appender File
Press any key to continue . . .

**Comment by shadowfacts** [comment]
Fixed in Forgelin 1.3.1 (~~currently pending approval on~~ [CurseForge](https://minecraft.curseforge.com/projects/shadowfacts-forgelin/files/2382547)).

**Comment by Morpheus1101** [comment]
This is still occuring using v1.3.1

**Comment by yrsegal** [comment]
that might be my fault

**Comment by yrsegal** [comment]
let me hot-release a new liblib with sorting after forgelin

**Comment by yrsegal** [comment]
try this @Morpheus1101 https://minecraft.curseforge.com/projects/librarianlib/files/2382603

**Comment by Morpheus1101** [comment]
Sadly no, still sais Forgelin dosent exist

[12:10:18] [Server thread/ERROR] [FML]: The mod librarianlib (LibrarianLib) requires mods [forgelin] to be available
[12:10:18] [Server thread/ERROR] [FML]: Mod librarianlib (LibrarianLib) requires [forgelin]
[12:10:18] [Server thread/ERROR]: Encountered an unexpected exception
net.minecraftforge.fml.common.MissingModsException: Mod librarianlib (LibrarianLib) requires [forgelin]
Missing Mods:
        forgelin : any


        at net.minecraftforge.fml.common.Loader.sortModList(Loader.java:256) ~[Loader.class:?]
        at net.minecraftforge.fml.common.Loader.loadMods(Loader.java:526) ~[Loader.class:?]
        at net.minecraftforge.fml.server.FMLServerHandler.beginServerLoading(FMLServerHandler.java:97) ~[FMLServerHandler.class:?]
        at net.minecraftforge.fml.common.FMLCommonHandler.onServerStart(FMLCommonHandler.java:328) ~[FMLCommonHandler.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:121) ~[ld.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:431) [MinecraftServer.class:?]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_112]

**Comment by yrsegal** [comment]
with the same error, with stars and all? because that code simply doesn't exist in the new version

**Comment by Morpheus1101** [comment]
Woops sorry forgot i had down graded FL .. let me try again

**Comment by Morpheus1101** [comment]
Server loads aok and loading client now, but i suspect client will be fine

**Comment by Morpheus1101** [comment]
Everything runs perfectly ok now :D
